deploy/kubernetes/overlays/windows/alpha/gcepd.yaml

```diff
@@ -0,0 +1,82 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-node
+spec:
+  template:
+    spec:
+      containers:
+        - securityContext:
+            $patch: delete
+          name: gce-pd-driver
+          image: gke.gcr.io/gcp-compute-persistent-disk-csi-driver-win
+          volumeMounts:
+            - name: kubelet-dir
+              mountPath: C:\var\lib\kubelet
+              mountPropagation: "None"
+            - name: plugin-dir
+              mountPath: C:\csi
+            - name: csi-proxy-disk-pipe
+              mountPath: \\.\pipe\csi-proxy-disk-v1alpha1
+            - name: csi-proxy-volume-pipe
+              mountPath: \\.\pipe\csi-proxy-volume-v1alpha1
+            - name: csi-proxy-filesystem-pipe
+              mountPath: \\.\pipe\csi-proxy-filesystem-v1alpha1
+            - name: kubelet-dir
+              mountPath: /var/lib/kubelet
+              mountPropagation: "Bidirectional"
+              $patch: delete
```

Comment:
Another possibility we could do so you don't need to delete all the linux paths, is:
1. Have the base yamls not include any linux specific stuff
2. Create linux overlay based on base to add the linux specifics
3. Base all the existing overlays on the linux overlay

Then Windows can build on top of the base overlay.

What do you think of that approach?

Reply:
I was thinking this too, and estimate there will be more changes and testing involved, so just go ahead with trying  this one first. 
Comparing these two, I do think the way you mentioned looks more clean. Let me try to update it with this approach..